### PR TITLE
py-nltk: Update

### PR DESCRIPTION
**Pull request by essandess**
py-nltk: Update

* Update to latest version
* Deleted older python3 versions

###### Type(s)

- [x] update
- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.x
Xcode 8.x

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-nltk/Portfile [hunk 1]
```diff
@@ -16,17 +16,24 @@ description         Natural Language Toolkit
...
+                    port:py${python.version}-setuptools
+    depends_lib-append \
+                    port:py${python.version}-click \
+                    port:py${python.version}-joblib \
+                    port:py${python.version}-matplotlib \
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-regex \
+                    port:py${python.version}-six \
+                    port:py${python.version}-tqdm \
+                    port:py${python.version}-yaml
```
Comment (reneeotten) [inline python/py-nltk/Portfile]: where do all these dependencies come from? I don't see most of them listed in upstream's ```setup.py``` for the released version (3.4.5 for which this update is). There is a largee list of dependency in the ```master``` branch, but these shouldn't be used.

File: python/py-nltk/Portfile [hunk 2]
```diff
@@ -4,8 +4,8 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           python 1.0
 
-github.setup        nltk nltk 3.4
-name                py-nltk
```
[the remark this answers is not in the document]
Comment (essandess) [reply python/py-nltk/Portfile]: Thanks done.

File: python/py-nltk/Portfile [hunk 3]
```diff
@@ -16,17 +16,24 @@ description         Natural Language Toolkit
 long_description    NLTK is Python modules for research and development in natural language processing
 homepage            http://www.nltk.org/
 
-python.versions     27 34 35 36 37 38
```
[the remark this answers is not in the document]
Comment (essandess) [reply python/py-nltk/Portfile]: Thanks done.

File: python/py-nltk/Portfile (hunk 1 above)
Comment (essandess) [reply python/py-nltk/Portfile]: https://github.com/nltk/nltk/blob/90d6779613fc698a39229e383f93b5c69bae672b/setup.py#L102-L109

Comment (reneeotten) [reply python/py-nltk/Portfile]: sure, but that's upstream's ```master``` branch and not for the released version

Comment (essandess) [reply python/py-nltk/Portfile]: > sure, but that's upstream's `master` branch and not for the released version

They'll appear in the next release soon enough.

I've been adding dependencies on more bleeding edge modules undergoing rapid development and have found that it's just more efficient to keep things running to anticipate the direction and toss in the kitchen sink.

Comment (reneeotten) [reply python/py-nltk/Portfile]: I completely disagree on this here and in other submissions! You can add these dependencies once upstream releases a version that requires them; I will personally not merge PRs in which this happens.

Comment (essandess) [reply python/py-nltk/Portfile]: Ok they're gone.

**Comment by reneeotten** [comment]
thanks @essandess  - merging under openmaintainer/maintainer timeout

**Pull request merged**